ci/testsettings.py

```diff
@@ -8,22 +8,16 @@
 
 DEBUG = False
 
-# include database settings to use Mariadb ver on production (5.5)
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.mysql",
+        "ENGINE": "django.db.backends.postgresql",
         "NAME": "ppa",
         "PASSWORD": "ppa",
-        "USER": "root",
+        "USER": "ppa",
         "HOST": "127.0.0.1",
-        "PORT": "3306",
-        "OPTIONS": {
-            # In each case, we want strict mode on to catch truncation issues
-            "init_command": "SET sql_mode='STRICT_TRANS_TABLES'",
```

Comment:
This was also unsupported by postgres (init_command).

Reply:
yes, this is mysql specific